app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,16 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self, nickname: str,
```

Comment:
After the opening bracket, write all the arguments on a new line; write the closing bracket on a new line too.

Reply:
Done